### DISCUSSION TITLE
Conversations webhooks and API returning error 404, but data going through

### DISCUSSION THREAD

**Discussion by Zornigs** [chatwoot/chatwoot#10172] [Q&A]
Hello!
Really liking using chatwoot here, i just have a small problem with it, being: I'm using the webhook to connect chatwoot with my whatsapp API and vice versa, the data goes through, meaning I recieve the data and chatwoot receives the data, I can have a functional chat. 
for example: 
On chatwoot:
![imagem_2024-09-26_113427190](https://github.com/user-attachments/assets/7c6acd2a-b0c6-401b-a67e-76be7eec2523)
On whatsapp
![imagem_2024-09-26_113456727](https://github.com/user-attachments/assets/e3a5b80d-6672-41fd-b881-163f0e96f99a)

The data flow is as follows:
Whatsapp -> N8N -> chatwoot API -> Live Chat
Chatwoot live chat -> Webhook to N8N -> whatsapp API

In the webhook data, there is this:
 },
      "additional_attributes": {},
      "content_attributes": {
        "external_error": "404 Not Found"
}

I'm using the self hosted version of chatwoot.
Any idea what this error could be and how to fix it?

**Comment by sojan-official** [comment]
Please check if you have any webhook integrations configured and it is working correctly. if the webhook throws error, it will be marked on the message. !!!

**Reply by Zornigs** [reply]
Yea, i do have it configured. Like i said, i receive the webhook data and can foward it to my whatsapp API using n8n.

**Reply by sojan-official** [reply]
ensure that your webhook responds with a success response for the request with 200 header

**Reply by Zornigs** [reply]
Hello!
Sorry for the delay in responding.
I'm sending the webhook response with a 200 status in the header and no info on the body, but the error still persists.
Do i need to send any other info with it?

**Reply by Zornigs** [reply]
Hello @sojan-official, is it just the header as 200? or is there another info i need to send?
Its Still returning me as 404 even if I send it on postman
